backend/backend.py

```diff
@@ -32,6 +33,20 @@ def __init__(self, width: int, height: int, publish_recording_status_func: Publi
         # Time between recording statuses, in seconds
         self.recording_status_interval = 60
 
+        # time that we last called self.send_camera_is_online
+        self.prev_camera_is_online_time = 0
+        # Time between recording statuses, in seconds
+        self.camera_is_online_interval = 60
+
+    @abstractmethod
+    def _is_camera_on(self) -> bool:
+        """ Checks if the camera / video feed / display is on
```

Comment:
The first line of a docstring should always end with a `.` and be in imperative mood.

Reply:
Done
f2bf442